tasks/main.yml

```diff
@@ -1,18 +1,18 @@
 ---
-- include: setup-RedHat.yml
-  when: ansible_os_family == 'RedHat'
-
-- include: setup-Debian.yml
-  when: ansible_os_family == 'Debian'
+- name: OS specific setup
+  include: "setup-{{ ansible_os_family }}.yml"
```

Comment:
I'm okay with the setup change (though to follow my role syntax style, please format it like this:

```yaml
# Setup/install tasks.
- include: "setup-{{ ansible_os_family }}.yml"
```

...but please remove the conditional (see reasoning elsewhere in this review).